docs/developers/developing_iree/profiling_with_tracy.md

```diff
@@ -72,58 +64,45 @@ For tracing the compiler, additionally set `IREE_ENABLE_COMPILER_TRACING` to
 `ON`. Compiler tracing is less stable, particularly on Linux with MLIR
 threading enabled (https://github.com/google/iree/issues/6404).
 
-## Permissions issues
 
-The profiled application (i.e. the Tracy client) needs to have appropriate
-permissions so perform the special I/O required to collect the profile
-information. This is OS-specific.
+## Building the Tracy Command Line Capture (the alternative "server")
 
-### Desktop Linux
+Optionally build the command capture tool with `-DIREE_BUILD_TRACY=ON `
 
-On desktop Linux, the profiled application must be run as root, e.g. with
-`sudo`. Otherwise, profile data will lack important components.
+```shell
+$ cmake \
+  -DIREE_BUILD_TRACY=ON \
+  -DIREE_ENABLE_RUNTIME_TRACING=ON \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  ... # other cmake arguments as usual
+```
 
-### Android
+The built command line tool will be found at `./build/tracy/iree-tracy-capture`
 
-On Android it is not necessary to run as root and in fact, Android graphical
-applications never run as root, so it's advisable to run all programs as
-non-root for consistency.
+## Building the Tracy UI (the "server")
 
-The Android device must be prepared as follows to enable Tracy profiling.
-* The device must be rooted.
-  * That means that in `adb shell`, the command `su` must succeed.
-  * That does NOT mean doing `adb root`. The effect of `adb root` is to have the
-    `adbd` daemon itself run as root, which causes `adb shell` to give you a
-    root shell by default. If you are in that case, consider doing `adb unroot`
-    to restart the `adbd` server as non-root. Not mandatory, but again, running
-    anything as root on Android is a deviation from normal user conditions.
-* Execute the following commands in a root shell on the device (i.e. `adb
-  shell`, then `su`, then the following commands). These are from the
-  [manual](#the-tracy-manual), but hard to find there, and copy-pasting from PDF
-  introduces unwanted whitespace. These settings normally persist until the next
-  reboot of the device.
-  * `setenforce 0`
-  * `mount -o remount,hidepid=0 /proc`
-  * `echo 0 > /proc/sys/kernel/perf_event_paranoid`
+This is explained in section 2.3 of the [manual](#the-tracy-manual) for Windows
```

Comment:
"section 2.3" is brittle - the manual could change sections. Can you give more context, like the section _name_ instead?

Reply:
Agreed. I was carrying this forward. It was been this way all along. https://github.com/bjacob/iree/commit/c51ae894111bef2f7241f13d7c02c89d8ad5601b . But let me try to find the section name and update.